src/stores/businessStore.ts

```diff
@@ -222,6 +223,18 @@ export const useBusinessStore = defineStore('business', {
       return (this.getBusinessState === EntityState.LIQUIDATION)
     },
 
+    /**
+     * Is False for non-BEN corps if FF is enabled.
+     * Is False for BENs and other entity types.
+     * Used to apply special pre-go-live functionality.
+     */
+    isDisableNonBenCorps (): boolean {
+      if (this.isBcCompany || this.isCcc || this.isUlc) {
+        return !GetFeatureFlag('enable-non-ben-corps')
+      }
+      return false
+    },
```

Comment:
This method name made more sense to me than `isEnableNonBenCorps`.

Reply:
No need to change. this looks good!